Sources/YBottomSheet/Animation/BottomSheetAnimator.swift

```diff
@@ -13,6 +13,13 @@ class BottomSheetAnimator: NSObject {
     /// Bottom sheet controller.
     let sheetViewController: BottomSheetController
     
+    /// Accessibility reduce motion is enabled or not.
+    var isReduceMotionEnabled: Bool {
+        get {
+            UIAccessibility.isReduceMotionEnabled
```

Comment:
|| UIAccessibility.prefersCrossFadeTransitions

and remove the `get { }` enclosure